src/main/java/graphql/schema/idl/SchemaPrinter.java

```diff
@@ -330,4 +359,40 @@ private void printType(PrintWriter out, GraphQLType type) {
         TypePrinter<Object> printer = printer(type.getClass());
         printer.print(out, type);
     }
+
+
+    private void printComments(PrintWriter out, Object graphQLType, String prefix) {
+        String description = getDescription(graphQLType);
+        if (description == null) {
+            return;
+        }
+        Stream<String> stream = Arrays.stream(description.split("\n"));
+        stream.map(s -> prefix + "#" + s + "\n").forEach(out::write);
+    }
+
+    private String getDescription(Object descriptionHolder) {
```

Comment:
Is there no common type here ?  GraphqlType perhaps?

I guess GraphQLEnumValueDefinition et all stop that